internal/util/idlocker.go

```diff
@@ -31,6 +31,9 @@ const (
 
 	// TargetPathOperationAlreadyExistsFmt string format to return for concurrent operation on target path.
 	TargetPathOperationAlreadyExistsFmt = "an operation with the given target path %s already exists"
+
+	// HostOperationAlreadyExistsFmt string format to return for concurrent operation.
```

Comment:
```suggestion
// HostOperationAlreadyExistsFmt string format to return for concurrent host operation.
```

comment seems to be too generic

Reply:
addressed this too.